packages/libs/react-ui/package.json

```diff
@@ -99,18 +89,16 @@
     "eslint-plugin-import": "~2.27.5",
     "eslint-plugin-storybook": "~0.6.13",
     "jest": "~29.6.1",
-    "jest-environment-jsdom": "~29.6.1",
     "jest-standard-reporter": "~2.0.0",
     "mini-css-extract-plugin": "2.7.6",
     "prettier": "~3.0.0",
-    "prettier-plugin-packagejson": "^2.4.5",
+    "knip": "~2.24.1",
```

Comment:
should this be here? I think it is only needed at the root level?

Reply:
Oh yeah I guess so